src/test/java/com/cruru/member/domain/repository/MemberRepositoryTest.java

```diff
@@ -32,7 +38,7 @@ void sameIdUpdate() {
         assertThat(findMember.getPhone()).isEqualTo("newPhoneNumber");
     }
 
-    @DisplayName("ID가 없는 사용자를 저장하면, 순차적으로 ID가 부여하여 저장된다.")
+    @DisplayName("ID가 없는 사용자를 저장하면, ID를 순차적으로 부여하여 저장된다.")
```

Comment:
진짜 별 거 아닌 거라 언급할지 말지 고민 좀 했는데 그냥 언급합니다. 
```suggestion
    @DisplayName("ID가 없는 사용자를 저장하면, ID를 순차적으로 부여하여 저장한다.")
```

Reply:
언급 감사합니다. 이런거 고민말고 언급해주세요!!